Description: Don't complain about multiple dots in filenames
 This patch prevents latex2html from complaining about multiple dots in
 filenames. Earlier versions of dvips were not able to handle this but the
 version on Debian is assumed to be safe.
Author: "Fred L. Drake, Jr." <fdrake@acm.org>
Bug-Debian: http://bugs-debian.org/114807
Bug-Debian: http://bugs-debian.org/118095

--- latex2html-2008-debian1.orig/latex2html.pin
+++ latex2html-2008-debian1/latex2html.pin
@@ -524,7 +524,7 @@ $TMP_PREFIX = "l2h" unless ($TMP_PREFIX)
 # This can be set to 1 when using a version of dvips that is safe
 # from the "dot-in-name" bug.
 # _TODO_ this should be determined by configure
-#$DVIPS_SAFE = 1;
+$DVIPS_SAFE = 1;
 
 $CHARSET = $charset || 'iso-8859-1';
 
@@ -671,7 +671,7 @@ sub set_if_false {
 
 sub check_for_dots {
     local($file) = @_;
-    if ($file =~ /\.[^.]*\./) {
+    if ($file =~ /\.[^.]*\./ && !$DVIPS_SAFE) {
 	die "\n\n\n *** Fatal Error --- but easy to fix ***\n"
 	    . "\nCannot have '.' in file-name prefix, else dvips fails on images"
 	    . "\nChange the name from  $file  and try again.\n\n";

